json_serializable/test/kitchen_sink_test.dart

```diff
@@ -237,8 +237,7 @@ Matcher _getMatcher(bool checked, String expectedKey, bool checkedAssignment) {
 
   if (checked) {
     if (checkedAssignment &&
-        const ['intIterable', 'datetime-iterable', 'validatedPropertyNo42']
-            .contains(expectedKey)) {
+        const ['intIterable', 'datetime-iterable'].contains(expectedKey)) {
```

Comment:
@jakemac53 – this is where the rubber hits the road. Now the expected key for `validatedPropertyNo42` is not `null`